examples/json-io/json_io.out

```diff
@@ -1,3 +1,5 @@
+# In the directory, which contains the `.bal` file, create a directory named `files`,
+# and add a JSON file named `jsonFile.json` in it.
```

Comment:
Creating the directory called `files` is enough. No need to put the JSON there, it will be automatically created.

Reply:
Fixed.